src/app/loadout-builder/filter/TierSelect.tsx

```diff
@@ -173,7 +184,7 @@ function MinMaxSelectInner({
         [opposite]:
           opposite === 'min' ? Math.min(stats[stat].min, value) : Math.max(stats[stat].max, value),
         ignored: false,
-      };
+      } as typeof update;
```

Comment:
I'm not fond of this but I can clean it up later.

Reply:
the conditionalness and inexpressible exclusiveness of the prop names ends up making typescript (reasonably) unsure they will exist. i hope ultimate fix involves this not using those capitalized strings

Reply:
Do you mean not using these?
```
const IGNORE = 'ignore';
const INCLUDE = 'include';
```

Reply:
Or the `StatTypes` types?